### DISCUSSION TITLE
On the creation of geojsons from input coordinates

### DISCUSSION THREAD

**Discussion by SimonFisher92** [SimonFisher92/Scottish_Snow_Satellite_Data#14] [General]
Hi everyone. It is now possible to download specific snow patches (in theory :) ), on the main branch. However, this needs some refinement. Im a little curious as to why the An Stuc patch image looks like this. 

![image](https://github.com/SimonFisher92/Scottish_Snow/assets/98879661/b99e7716-d073-409c-884b-40886124bc5a)

Clearly it is not restricted enough to the patch (in terms of zoom), but curiously, it is not centred over it either!

**Comment by SimonFisher92** [comment]
The top left and bottom right of the geojson sit here:

![image](https://github.com/SimonFisher92/Scottish_Snow/assets/98879661/7c9f36fe-5b3c-47eb-8184-2acc31e47487)

Also, for some reason on google, I had to swap lat/lon or else you end up in the Indian Ocean (as Murray observed)

**Comment by SimonFisher92** [comment]
FYI this command will allow you to fetch An_Stuc data (or any other patch for that matter)

 python .\main.py --data_dir "../../data/snow_patches" --geojson_path "../get_patches/An_Stuc.geojson" --api_user "YOURS" --api_password "YOURS"

**Comment by SimonFisher92** [comment]
Checking the geojson online, it looks pretty much bang-on for the AnStuc patch, so I dont think this is a geojson problem

![image](https://github.com/SimonFisher92/Scottish_Snow/assets/98879661/5acc5211-6e5e-4477-a093-86986f8469c5)

**Comment by SimonFisher92** [comment]
Sorry for the spam, that should be the issue with the geojsons fixed on branch 9-imagebased_segmentation

![image](https://github.com/SimonFisher92/Scottish_Snow/assets/98879661/3e796d02-ed83-4cf0-b95f-a0ec40fe04a4)

**Comment by murraycutforth** [comment]
Great stuff
